readthedocs/tutorial.rst

```diff
@@ -410,18 +419,14 @@ Below are a few more examples of ``slurm submit``:
       submit --stdout /home/xenon/ls.stdout.txt ls -- -la
 
       # check to see if the output was written to file /home/xenon/ls.stdout.txt
-      ssh -p 10022 xenon@localhost ls -l
-      # see what's in it
-      ssh -p 10022 xenon@localhost cat ls.stdout.txt
+      xenon filesystem sftp --location localhost:10022 --username xenon --password javagat download ls.stdout.txt -
```

Comment:
I would be in favor of retaining ``ssh`` here, since you're comparing the xenon result against what people already know.

Reply:
Nope. I've just explained to them how to do this a few minutes earlier. Switching back to ssh shows a lack of trust in our own tool.